### PR TITLE
DEX-1201 swap out match_phrase for the more efficient term, which als…

### PR DESCRIPTION
**Pull request by Atticus29**
…o reflects what is actually being used in the front end

- Ensure that the PR uses a consolidated database migration

N/A

  - Example: One database migration is proposed (revision `<insert develop revision>` -> `<insert new revision>`)
- Ensure that the PR is properly sanitized
  - Example: No sensitive data or large content was added to this PR
 
N/A

### CODE REVIEW

File: tests/modules/sightings/resources/test_identify_sighting.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def test_sighting_identification(
                                 'minimum_should_match': 1,
                                 'should': [
                                     {
-                                        'match_phrase': {
+                                        'term': {
```
Comment (Atticus29) [inline tests/modules/sightings/resources/test_identify_sighting.py:178]: Not sure about this one.

File: tests/modules/sightings/resources/test_sighting_elasticsearch.py [hunk 2]
```diff
@@ -157,8 +157,8 @@ def test_search_with_elasticsearch(
         'bool': {
             'minimum_should_match': 1,
             'should': [
-                {'match_phrase': {'guid': {'query': searchTerm}}},
-                {'match_phrase': {'locationId_id': {'query': searchTerm}}},
+                {'term': {'guid': searchTerm}},
+                {'term': {'locationId_id': searchTerm}},
```
Comment (Atticus29) [inline tests/modules/sightings/resources/test_sighting_elasticsearch.py:161]: Same uncertainty.

File: tests/modules/sightings/resources/test_identify_sighting.py (hunk 1 above)
Comment (bluemellophone) [reply tests/modules/sightings/resources/test_identify_sighting.py:178]: Yes, `locationId` is a keyword

**Review by bluemellophone — APPROVED** [review]
LGTM

**Pull request merged**